### DISCUSSION TITLE
How to enable automatic import similar to app in module?

### DISCUSSION THREAD

**Discussion by gzf6** [nuxt/nuxt#25300] [Questions]
I hope to use auto import in the business module project.

**Comment by xjccc** [comment]
[Auto-imports](https://nuxt.com/docs/guide/concepts/auto-imports)

**Reply by gzf6** [reply]
I mean in standalone module project, not in app. For example, when using useI18n in a module, you must import it from vue-i18n.

**Reply by xjccc** [reply]
read more docs like this?
![image](https://github.com/nuxt/nuxt/assets/16478776/c5aa0889-4fe5-442b-b2a6-d77cb124542c)

**Reply by gzf6** [reply]
We may not be discussing the same thing. You can try it with `npx nuxi init -t module my-module`, and install @nuxtjs/i18n. Try using useI18n in custom component.

**Reply by xjccc** [reply]
use auto-import in a module?
[auto-import](https://nuxt.com/docs/api/kit/autoimports#auto-imports)

**Reply by gzf6** [reply]
Yes, use in a module ,not in app. 
The document only describes that module provides automatic import function to app. Now I want to use automatic import in module.
